docs/frontend/40-git.rst

```diff
@@ -1,22 +1,133 @@
 Git
----
-
-Create a feature branch and push to Redmine for review.
+===
 
 .. _git-deployment:
 
-Deployment
-~~~~~~~~~~
+.. seealso::
+    Public elements - :ref:`github-git`
+
+Pull Requests (PRs)
+~~~~~~~~~~~~~~~~~~~
+
+.. seealso:: https://help.github.com/articles/proposing-changes-to-your-work-with-pull-requests/
+
+Pull Requests are the most efficient way to safely review and merge code.
+To create a Pull Request, create a new branch (based off ``master``)::
+
+    $ git checkout master
+    $ git pull
+    $ git checkout -b my-feature
+
+Develop the feature and push the branch to GitHub::
+
+    $ git push --set-upstream origin my-feature
+
+Then head to https://github.com/Neovici/cosmoz-frontend/branches/yours and find your newly created branch,
+and press the :guilabel:`New pull request` button.
+Make sure the PR base is the ``master`` branch and describe the PR in the form.
+Assign yourself the PR, and if known, assign one or more reviewers, labels and projects.
```

Comment:
Maybe this is to "vague".
If we want to use more PRs, we probably need clear rules regarding reviewers.

A suggestion:
* Most of the time, assign the team "Neovici/frontend" as the unique reviewer
This will mean you expect a review from a single member of the team. 
Anyone in the team can review the PR, and if approved, then the reviwer should merge.
* If you think the PR should be reviewed by specific persons, then assign these persons.
The last reviewer should merge the PR.

What do you think ?

Reply:
Sounds good! Would you have time to add this suggestion, fix the spelling error and merge this pr? 🙏